sub-packages/bionemo-geneformer/tests/bionemo/geneformer/conftest.py

```diff
@@ -18,6 +18,7 @@
 
 import pytest
 
+from bionemo.core.data.load import load as core_load
 from bionemo.testing.data.load import load
```

Comment:
I am not sure what it is, probably load method from core. You can remove load from testing

https://github.com/NVIDIA/bionemo-framework/blob/656c70acb584bb0366faab61fa3a2218e6ed26e7/sub-packages/bionemo-testing/src/bionemo/testing/data/load.py#L17